src/helpers.js

```diff
@@ -23,28 +23,32 @@ const getFileExtension = (fileName) => {
   return (i < 0) ? '' : fileName.substr(i + 1);
 };
 
-const organize = (spinner, source, output, fileName, type) => {
+const organize = (spinner, source, output, fileName, type, listOnly) => {
   mkdir(output);
 
   const typeDir = path.resolve(output, type);
   mkdir(typeDir);
 
   return new Promise((resolve, reject) => {
-    mv(path.resolve(source, fileName), path.resolve(typeDir, fileName), (err) => {
-      if (err) {
-        const errorMessage = `Couldn't move ${fileName} because of following error: ${err}`;
-        spinner.warn(errorMessage);
-        reject(new Error(errorMessage));
-      } else {
-        const successMessage = `Moved ${fileName} to ${type} folder`;
-        spinner.info(successMessage);
-        resolve(successMessage);
-      }
-    });
+    if (listOnly) {
+      console.log('mv '+ path.resolve(source, fileName) + ' ' + path.resolve(typeDir, fileName));
```

Comment:
@uditarora We need to resolve promise here. This message can be passed in resolve.